.github/actions/prepare-rtl/action.yml

```diff
@@ -0,0 +1,41 @@
+name: prepare-rtl
+description: 'Builds RTL based on parameters, caches the entire chipyard root dir when done'
+
+inputs:
+  tools-version:
```

Comment:
Same as other action, this isn't needed since caching is done outside of this action.